### ISSUE TITLE
PostGen Modal Keyboard Nav Button Missing

### ISSUE THREAD

**Issue by trevorNgo** [Microsoft/WebTemplateStudio#500]
![image](https://user-images.githubusercontent.com/23327878/56330303-509e6280-613c-11e9-80ed-9434bc939f89.png)
Tabbing does not select the close wizard button
